src/components/sidebar/components/sidebar-nav/sidebar-nav-menu-item/sidebar-nav-menu-item.module.css

```diff
@@ -7,7 +11,8 @@
   color: var(--token-color-palette-neutral-500);
   display: flex;
   font-size: 14px;
-  font-weight: 400;
+  font-weight: var(--token-typography-font-weight-regular);
+  gap: 8px;
```

Comment:
This `gap` property ensures there is space between the item's text and icon (if present), which can be an issue for long text on small viewports.

Reply:
This should already be handled with the `margin-right` property inside of the `navMenuItemLabel` class defined lower in this file. 😊

Reply:
Ah cool - I'd started to try to reduce some repetition in `SidebarNavLink` but then reverted my changes, forgot to fix this 👍 ty 💖